setup.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+from os import path
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
+# To use a consistent encoding
+
+BASE_DIR = path.abspath(path.dirname(__file__))
+
+with open(path.join(BASE_DIR, 'README.md')) as f:
+    LONG_DISCRIPTION = f.read()
+
+setup(
+    name='profile-manager',
+
+    version='0.1',
+
+    description='A sample Python project',
```

Comment:
We can do better here, how about "Process and route signals according to certain rules"